### ISSUE TITLE
Spectral library axis flipped

### ISSUE THREAD

**Issue by ecoinfo-netra** [EnMAP-Box/enmap-box#916]
Dear Team,
I am using the Tyrol_Austria_L2A example dataset from Enmap. 
Everytime I try to create a spectral library, my axis are flipped i.e. x axis is band number and y axis is wavelength.
I have installed and reinstalled the enmap box a couple of times and I still get the same problem.

When I use the default example data from enmap box the axis are fine. 

I am using the version 3.13.3. 
Enmap Tyrol (please see x axis in the spectral library)
<img width="826" alt="enmap_tyrol" src="https://github.com/EnMAP-Box/enmap-box/assets/92730392/9ae4432f-88f7-4b43-95cf-8e5ebf90ecab">

EnMap box default example Potsdam
<img width="818" alt="enmap_default_example" src="https://github.com/EnMAP-Box/enmap-box/assets/92730392/941b0ee6-4e43-4d64-a9fa-792b64d37ab9">

**Comment by janzandr** [comment]
1. Your plot is not correct, even with the Potsdam dataset:
![image](https://github.com/EnMAP-Box/enmap-box/assets/45030715/1589c065-bd65-491b-99de-9dc8a322a428)
2. I can't reproduce the problem on my system.
3. Before we start narrowing down the problem, please make sure that you are using a recent QGIS 3.34+ together with the latest  EnMAP-Box v3.14, and try again.

**Comment by janzandr** [comment]
You need to import the EnMAP product:
Menu > Project > Add product > EnMAP L2A

**Comment by janzandr** [comment]
@ecoinfo-netra, sorry, I just realized, that you did use the product import algorithm.
The error you show is a known issue, which is caused by the very long file path:
![image](https://github.com/EnMAP-Box/enmap-box/assets/45030715/c309bd4b-4dc6-43bf-9d59-cb5f2a7457e4)

You have to move the product folder, e.g. to C:\data\<product folder>

**Comment by ecoinfo-netra** [comment]
Yes, thank you. Now it works.

**Comment by janzandr** [comment]
Duplicate of https://github.com/EnMAP-Box/enmap-box/issues/595.
